graphblas.sh

```
@@ -0,0 +1,7 @@
+#!/bin/bash
+git clone --branch v10.1.1 --single-branch https://github.com/DrTimothyAldenDavis/GraphBLAS.git
+cd GraphBLAS
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)